### PR TITLE
fix: add workspace tag

### LINKED ISSUE #23: Add workspace tag
Add a tag to identify which resources are deployed as part of which Terraform Workspace

### PR DESCRIPTION
**Pull request by chriscasto**
Fixes #23

### CODE REVIEW

File: bloom-instance/main.tf [hunk 1]
```diff
@@ -19,13 +19,16 @@ provider "aws" {
 }
 
 locals {
-  default_name = "${var.resource_prefix}-${var.sdlc_stage}-${var.application_name}"
+  # We may want to rearrange the order of these in the future based on how easy
+  # or hard it is to read at a glance in the console.
+  default_name = "${var.resource_prefix}:${terraform.workspace}-${var.sdlc_stage}:${var.application_name}"
```
Comment (lisunshiny) [inline bloom-instance/main.tf:24]: @chriscasto can you add a test plan screenshot? we should be able to see the output of `terraform plan` change I think?

Comment (lisunshiny) [reply bloom-instance/main.tf:24]: (also I approve of the `-` to `:` switch)

**Review by lisunshiny — CHANGES_REQUESTED** [review]

File: bloom-instance/main.tf (hunk 1 above)
Comment (chriscasto) [reply bloom-instance/main.tf:24]: You got it

Screenshot 2023-02-21 at 4 47 29 PM

**Review by lisunshiny — APPROVED** [review]
ty! lgtm

**Pull request merged**